### ISSUE TITLE
How can I get the paper`s supplementary material?

### ISSUE THREAD

**Issue by Cloveryww** [DrSleep/light-weight-refinenet#35]
Hi!
I am reproducing the MobilenetV2`s performance in PascalVOC but suffer from some difficults(the model`s performance  do not get better with training, the performance in val set is very bad--all classes(except background)`s iou is zero).So, could you please give more detailed info about the trainning strategy？
Thanks

**Comment by DrSleep** [comment]
Make sure that you initialise MBv2 from ImageNet weights, and tune the learning rate (or try the setup we used in the paper).

**Comment by Cloveryww** [comment]
Thanks, I have solved the problem.
